core-web/libs/dot-primeng-theme-styles/src/scss/dotcms-theme/components/_tag.scss

```diff
@@ -12,27 +12,56 @@
     border-radius: $tag-radius-md;
 }
 
-.p-tag.p-tag-success {
-    background-color: $color-alert-green-light;
-    color: $color-alert-green;
+.p-tag .p-tag-icon {
+    margin-right: $spacing-1;
+    font-size: $font-size-x-small;
 }
 
-.p-tag.p-tag-info {
-    background-color: $color-alert-blue-light;
-    color: $color-alert-blue;
-}
+.p-tag {
+    // defaults statuses
+    &.p-tag-success {
+        background-color: $color-alert-green-light;
+        color: $color-alert-green;
+    }
 
-.p-tag.p-tag-warning {
-    background-color: $color-alert-yellow-light;
-    color: $color-alert-yellow;
-}
+    &.p-tag-info {
+        background-color: $color-alert-blue-light;
+        color: $color-alert-blue;
+    }
 
-.p-tag.p-tag-danger {
-    background-color: $color-alert-red-light;
-    color: $color-alert-red;
-}
+    &.p-tag-warning {
+        background-color: $color-alert-yellow-light;
+        color: $color-alert-yellow;
+    }
 
-.p-tag .p-tag-icon {
-    margin-right: $spacing-1;
-    font-size: $font-size-x-small;
+    &.p-tag-danger {
+        background-color: $color-alert-red-light;
+        color: $color-alert-red;
+    }
+
+    // Experiments statuses
+    &.running {
+        color: $color-accessible-text-green;
+        background-color: $color-accessible-text-green-bg;
+    }
+
+    &.draft {
+        color: $color-accessible-text-yellow;
+        background-color: $color-accessible-text-yellow-bg;
+    }
+
+    &.scheduled {
+        color: $color-accessible-text-purple;
+        background-color: $color-accessible-text-purple-bg;
+    }
+
+    &.ended {
+        color: $color-accessible-text-blue;
+        background-color: $color-accessible-text-blue-bg;
+    }
+
+    &.archived {
+        color: $color-accessible-text-gray;
```

Comment:
This is only for experiments not all the tags component for that should not be in this file.